include/Dialect/QUIR/Transforms/BreakReset.h

```diff
@@ -49,10 +58,23 @@ struct BreakResetPass
       llvm::cl::desc("Number of cycles of delay to add between reset "
                      "iterations, default is 1000"),
       llvm::cl::value_desc("num"), llvm::cl::init(1000)};
+  Option<bool> putCallGatesAndMeasuresIntoCircuit{
+      *this, "quantum-gates-in-circuit",
+      llvm::cl::desc("an option to put call gates and measures into circuit"),
+      llvm::cl::value_desc("bool"), llvm::cl::init(false)};
```

Comment:
Should this default to true rather than false? I would expect that we would want to put the gates into circuits as standard practice and then have this option to disable if required for testing.